app/models/flood_risk_engine/enrollment.rb

```diff
@@ -20,6 +19,8 @@ class Enrollment < ActiveRecord::Base
     # The Correspondence Contact Details related to this Application a.k.a Main Contact
     belongs_to :correspondence_contact, class_name: "Contact"
 
+    has_one :location, as: :locatable, dependent: :restrict_with_exception
```

Comment:
At the moment, the only requirement is that the location that the exemptions will refer to have a grid reference and easting/northing. All exemptions must refer to a single location (that part of the spec. see: https://eaflood.atlassian.net/browse/RIP-60). However, `exemption_location` may be a better name.